prebid-mobile/dr-prebid.md

```diff
@@ -62,7 +62,7 @@ These settings will enable you to test whether the Prebid Mobile line item in yo
 The primary ad server you’re using to serve your ads.
 
 Select from:
--	*DFP*
+-	*Google Ad Manager*
```

Comment:
This is an option in the UI. You should check with the PBM team to see if they've updated this - if not we need to leave as DFP to match the UI.